File: AndroidCinema/app/src/main/java/com/crpr/androidcinema/domain/welcome_wizard/WelcomeWizardResult.java
package com.crpr.androidcinema.domain.welcome_wizard;

import com.crpr.androidcinema.domain.common.Result;

/**
 * Created by claudioribeiro on 14/07/16.
 */
public class WelcomeWizardResult extends Result {

    private boolean isWelcomeWizardDone;

    public WelcomeWizardResult(int statusCode, boolean isWelcomeWizardDone) {
        super(statusCode);
        this.isWelcomeWizardDone = isWelcomeWizardDone;
    }

    public boolean isWelcomeWizardDone(){
        return this.isWelcomeWizardDone;
    }
}

File: AndroidCinema/app/src/main/java/com/crpr/androidcinema/domain/welcome_wizard/WelcomeWizardResult.kt
package com.crpr.androidcinema.domain.welcome_wizard

import com.crpr.androidcinema.domain.common.Result

/**
 * Created by claudioribeiro on 14/07/16.
 */
class WelcomeWizardResult : Result {

    var isWelcomeWizardDone: Boolean = false

    constructor(statusCode: Int, isWelcomeWizardDone: Boolean) : super(statusCode) {
        this.isWelcomeWizardDone = isWelcomeWizardDone
    }

    constructor(statusCode: Int, errorMessage: String) : super(statusCode, errorMessage) {}
}
